firstvoices/backend/management/commands/copy_site.py

```diff
@@ -0,0 +1,532 @@
+
+            copy_related_media(page, source_media, audio_map, image_map, video_map)
+            page.save()
+
+
+def copy_dictionary_entries(
+    source_site,
+    target_site,
+    dictionary_entry_map,
+    category_map,
+    character_map,
+    audio_map,
+    image_map,
+    video_map,
+):
+    dictionary_entries = (
+        DictionaryEntry.objects.filter(site=source_site)
+        .annotate(related_entry_count=Count("related_dictionary_entries"))
+        .order_by("related_entry_count")
+        .distinct()
+    )
+    for entry in dictionary_entries:
+        source_categories = list(entry.categories.all())
+        source_related_characters = list(entry.related_characters.all())
+        source_related_entries = list(entry.related_dictionary_entries.all())
+        source_media = {
+            "audio": list(entry.related_audio.all()),
+            "images": list(entry.related_images.all()),
+            "videos": list(entry.related_videos.all()),
+        }
+
+        source_entry_id = entry.id
+        target_entry_id = uuid.uuid4()
+        dictionary_entry_map[source_entry_id] = target_entry_id
+
+        entry.id = target_entry_id
+        entry.site = target_site
+        entry.batch_id = ""
+        entry.import_job = None
+        entry.save()
+
+        updated_categories = [
+            Category.objects.get(id=category_map[category.id])
+            for category in source_categories
+        ]
+        entry.categories.set(updated_categories)
+
+        updated_related_characters = [
+            Character.objects.get(id=character_map[char.id])
+            for char in source_related_characters
+        ]
+        entry.related_characters.set(updated_related_characters)
+
+        updated_related_entries = [
+            DictionaryEntry.objects.get(
+                id=dictionary_entry_map[source_related_entry.id]
+            )
+            for source_related_entry in source_related_entries
+        ]
+        entry.related_dictionary_entries.set(updated_related_entries)
+
+        copy_related_media(entry, source_media, audio_map, image_map, video_map)
+        entry.save()
+
+
+def copy_immersion_labels(source_site, target_site, dictionary_entry_map):
+    imm_labels = ImmersionLabel.objects.filter(site=source_site)
+    for imm_label in imm_labels:
+        imm_label.id = uuid.uuid4()
+        imm_label.site = target_site
+
+        source_dictionary_entry = imm_label.dictionary_entry
+        target_dictionary_entry = DictionaryEntry.objects.get(
+            id=dictionary_entry_map[source_dictionary_entry.id]
+        )
+        imm_label.dictionary_entry = target_dictionary_entry
+
+        imm_label.save()
+
+
+def copy_related_objects(source_site, target_site, logger):
+    image_map = {}
+    video_map = {}
+    audio_map = {}
+    category_map = {}
+    character_map = {}
+    dictionary_entry_map = {}
+
+    copy_site_features(source_site, target_site)
+    copy_site_menu(source_site, target_site)
```

Comment:
If we're not copying pages, we probably don't want to copy the menu either. Most customizations are done to include custom pages in the menu